### PR TITLE
Added a note to RPC runtime API

### PR DESCRIPTION
**Pull request by ToriLindsay**
### Summary

#15256

### CODE REVIEW

File: src/content/docs/workers/runtime-apis/rpc/index.mdx [hunk 1]
```diff
@@ -9,6 +9,10 @@ description: The built-in, JavaScript-native RPC system built into Workers and
 
 import { DirectoryListing, Render } from "~/components"
 
+:::note
+If you are adding this feature to existing Workers or Pages, Cloudflare recommends [updating the Workers’ or Pages’ compatibility date](/workers/configuration/compatibility-dates) first.
```
Comment (dario-piotrowicz) [inline src/content/docs/workers/runtime-apis/rpc/index.mdx]:
```suggestion
If you are adding this feature to existing Workers or Pages, please make sure to have the correct [compatibility date](/workers/configuration/compatibility-dates) set.
```

Comment (dario-piotrowicz) [reply src/content/docs/workers/runtime-apis/rpc/index.mdx]: Or maybe we could/should just say something along these lines:
```
:::caution 
 
	 To use RPC, you must define a compatibility date of `2024-04-03` or higher, or include `rpc` in your [compatibility flags](/workers/configuration/compatibility-dates/#nodejs-compatibility-flag).
	  
::: 
```

(to stay compatible with other pages such as https://github.com/cloudflare/cloudflare-docs/blob/f2b7164c163beba535ffb4cedb7a6250030b95bf/src/content/docs/workers/testing/vitest-integration/configuration.mdx?plain=1#L166-L172)

🤔

Comment (ToriLindsay) [inline src/content/docs/workers/runtime-apis/rpc/index.mdx]:
```suggestion
To use RPC, [define a compatibility date](/workers/configuration/compatibility-dates) of `2024-04-03` or higher, or include `rpc` in your [compatibility flags](/workers/configuration/compatibility-dates/#nodejs-compatibility-flag).
```

Comment (ToriLindsay) [reply src/content/docs/workers/runtime-apis/rpc/index.mdx]: Updated! (I prefer your second suggestion, as it's more specific and direct)

**Review by dario-piotrowicz — APPROVED** [review]
Looks good to me 😄

File: src/content/docs/workers/runtime-apis/rpc/index.mdx (hunk 1 above)
Comment (dario-piotrowicz) [reply src/content/docs/workers/runtime-apis/rpc/index.mdx]: Cool! thanks for updating it 🫶

**Review by Oxyjun — APPROVED** [review]

File: src/content/docs/workers/runtime-apis/rpc/index.mdx [hunk 2]
```diff
@@ -9,6 +9,10 @@ description: The built-in, JavaScript-native RPC system built into Workers and
 
 import { DirectoryListing, Render } from "~/components"
 
+:::note
+To use RPC, [define a compatibility date](/workers/configuration/compatibility-dates) of `2024-04-03` or higher, or include `rpc` in your [compatibility flags](/workers/configuration/compatibility-dates/#nodejs-compatibility-flag).
```
Comment (irvinebroque) [inline src/content/docs/workers/runtime-apis/rpc/index.mdx:13]: Why does this link to the node.js compat flag section?

**Pull request merged**